sound/soc/sof/pcm.c

```diff
@@ -733,11 +733,11 @@ static int sof_pcm_probe(struct snd_soc_component *component)
 	}
 
 	/*
-	 * Some platforms in SOF, ex: BYT, may not have their platform PM
-	 * callbacks set. Increment the usage count so as to
-	 * prevent the device from entering runtime suspend.
+	 * Some boards such as the Minnowboard do not support PM.
+	 * Increment the usage count so as to prevent the device
+	 * from entering runtime suspend.
```

Comment:
For MinnowBoards, isn't it suspend-resume that's broken for audio?
if you use rtcwake -m mem -s 3, would audio work when you resume?

If not, then pm_runtime doesn't help.

Reply:
@plbossart if I remember correctly, on the minnowboards, not holding the ref count to prevent runtime PM somehow ends up in the DSP getting powered off. I might have to test this again though.